demo/svg-sample-icons.js

```diff
@@ -1,8 +1,21 @@
+/**
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
+
 import '../iron-iconset-svg.js';
+
 const $_documentContainer = document.createElement('template');
 $_documentContainer.setAttribute('style', 'display: none;');
 
-$_documentContainer.innerHTML = `<iron-iconset-svg name="svg-sample-icons" size="100">
+$_documentContainer.innerHTML =
+    `<iron-iconset-svg name="svg-sample-icons" size="100">
```

Comment:
`html` function here too

Reply:
This was the one you were mentioning offline that probably shouldn't use `html`, right? (left unchanged)

Reply:
Or, maybe you were talking about [this comment](https://github.com/PolymerElements/iron-icons/pull/88#discussion_r203135953) instead?